proxyclient/m1n1/proxy.py

```diff
@@ -131,13 +131,20 @@ class UartInterface(Reloadable):
     REPLY_LEN = 36
     EVENT_HDR_LEN = 8
 
+    DEFAULT_UART_DEV="/dev/m1n1"
+    DEFAULT_BAUD_RATE="115200"
+    if platform.system() == 'Linux':
+        DEFAULT_UART_DEV="/dev/ttyACM0"
```

Comment:
This changes the existing behavior. The idea is that on Linux you install the udev rules in `udev/`, which gives you the `/dev/m1n1` symlink. Maybe we can just leave Linux at the default?

Reply:
Yes fine with me to keep Linux as default